### PR TITLE
Fix enumerated list

### PR DESCRIPTION
**Pull request by Landerstraeten**
Hardcode the list instead of auto-enumeration

### CODE REVIEW

**Comment by xabbuh** [comment]
I don't see why we would want to do that. Not enumerating items statically has the advantage than we can delete/add items in the middle without having to adjust all other items.

**Comment by Landerstraeten** [comment]
@xabbuh, check http://symfony.com/doc/master/bundles/FOSRestBundle/versioning.html
It's 3 times '1'...

File: Resources/doc/versioning.rst [hunk 1]
```diff
@@ -77,17 +77,17 @@ automatically in the ``ViewHandler``.
 
 If you want to version by Accept header, you will need to do the following:
 
-#. The format listener must be enabled
+1. The format listener must be enabled
 
 See :doc:`Format Listener <format_listener>`
```
Comment (xabbuh) [inline Resources/doc/versioning.rst]: We need to indent the content of the list items by three spaces each. That should fix the rendering of the list.

Comment (Landerstraeten) [reply Resources/doc/versioning.rst]: Oh ok. I'll update this.

**Comment by xabbuh** [comment]
I see. Thanks for the hint. Actually, we currently have three lists with one item each. Indenting the other paragraphs of the items will solve that (see my inline comment above).

**Comment by Landerstraeten** [comment]
@xabbuh, are you happy with the change?

**Pull request merged**